ui/src/components/common/CustomAlert/CustomAlert.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import './CustomAlert.css';
+import PropTypes from 'prop-types';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+
+/**
+ * A custom alert component that wraps the Patternfly 4 Alert.
+ *
+ * @param {Object} visible - The visible state of the alert.
+ * @param {func} onClose - The callback function to execute on close.
+ * @param {*} children - Any child props - usually the description.
+ * @param {Object} props - The component props
+ */
+const CustomAlert = ({ visible, onClose, children, ...props }) => {
+  const [isVisible, setVisibility] = useState(visible);
+
+  useEffect(() => {
+    setVisibility(visible);
+  }, [visible]);
+
+  if (!isVisible) {
```

Comment:
```suggestion
  if (! visible) {
```
Lines 15 - 20 do not look to be needed. You can switch the conditional to just use the suggestion above.

Reply:
Yeah, I don't know why I thought I needed this :joy:

Reply:
was probably needed with the timeout logic.